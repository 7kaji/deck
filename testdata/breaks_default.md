# Testing Soft Line Breaks (Default)

This is a test
with soft line break
that should NOT render as breaks by default.

---

## List with soft breaks

- First item with
  soft break
- Second item with
  another soft break

---

## Blockquote with breaks

> This is a quote
> with soft line breaks
> in multiple places.
