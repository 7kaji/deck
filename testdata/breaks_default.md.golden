[
  {
    "layout": "",
    "titles": [
      "Testing Soft Line Breaks (Default)"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a test "
              },
              {
                "value": "with soft line break "
              },
              {
                "value": "that should NOT render as breaks by default."
              }
            ]
          }
        ]
      }
    ]
  },
  {
    "layout": "",
    "titles": [
      "List with soft breaks"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "First item with "
              },
              {
                "value": "soft break"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Second item with "
              },
              {
                "value": "another soft break"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  },
  {
    "layout": "",
    "titles": [
      "Blockquote with breaks"
    ],
    "block_quotes": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a quote "
              },
              {
                "value": "with soft line breaks "
              },
              {
                "value": "in multiple places."
              }
            ]
          }
        ]
      }
    ]
  }
]